lib/compat/wordpress-6.8/site-editor.php

```diff
@@ -130,7 +130,7 @@ function gutenberg_styles_wp_die_handler( $default_handler ) {
  * @global array $submenu
  */
 function gutenberg_add_styles_submenu_item() {
-	if ( ! wp_is_block_theme() && ( current_theme_supports( 'editor-styles' ) || wp_theme_has_theme_json() ) ) {
+	if ( ! wp_is_block_theme() ) {
```

Comment:
I'll leave @tellthemachines to chime in with a more educated review, but I was just wondering what the implication of this comment from https://github.com/WordPress/gutenberg/pull/66851#discussion_r1872695946 would be:

> because if the theme has neither, the style book will be useless.


Does these mean if a theme doesn't have editor or theme.json styles, the style book would just render the default styles?

Reply:
> Does these mean if a theme doesn't have editor or theme.json styles, the style book would just render the default styles?

That's what I thought at first too. But as mentioned in this PR, it seems there's a way to apply styles for blocks on the editor side other than theme.json or editor-styles support.

Reply:
I'm not convinced we should do this, as having a condition means themes have the choice to opt in or out. If themes really want the stylebook, it costs nothing to add theme support for editor-styles. Twenty Twenty has it even though it enqueues its editor styles via `enqueue_block_assets` as in your example.